querydsl-core/src/main/java/com/querydsl/core/support/SerializerBase.java

```diff
@@ -170,8 +168,12 @@ public void setAnonParamPrefix(String prefix) {
         this.anonParamPrefix = prefix;
     }
 
+    /**
+     * Not used anymore
```

Comment:
Maybe we should elaborate a little more here and explain why the `normalize` flag isn't used anymore and what the new behavior is so users know what's going on when they upgrade.